jest.config.js

```diff
@@ -24,13 +24,13 @@ module.exports = {
     './src/js/state/**/*.js': {
       statements: 100,
     },
-    './src/js/components/': {
-      branches: 70,
-      statements: 70,
+    './src/js/components/Graphs/': {
```

Comment:
Maybe rather add some tests? :P

Reply:
I want, but we have some problems with this component testing.  Jest cannot render canvas in virtual dom. We don't want to block this and another PR, so we changed coverage for this component. Tomek know about this and accept this "solution". We have an issue for that on github.

Reply:
@mikolajkozakiewicz can you do the "second" check? any ideas how to fix the issue with canvas?

Reply:
Have you tried https://github.com/jsdom/jsdom#canvas-support ?
Jest underneath uses jsdom, so maybe this one would help